README.md

```diff
@@ -0,0 +1,27 @@
+tidb-inspect-tools
+------
+
```

Comment:
It would be better if we put a brief introduction to the `tidb-inspect-tools` here. Perhaps one or two sentences.